bundles/org.openhab.binding.kodi/README.md

```diff
@@ -4,13 +4,13 @@
 Kodi runs on Linux, OS X, BSD, Windows, iOS, and Android.
 It allows users to play and view most videos, music, podcasts, and other digital media files from local and network storage media and the internet.
 
-The Kodi Binding integrated Kodi media center support with openHAB, allowing both controlling the player as well as retrieving player status data like the currently played movie title.
+The Kodi Binding integrates Kodi media center support with openHAB, allowing both controlling the player as well as retrieving player status data like the currently played movie title.
 
 The Kodi binding is the successor to the openHAB 1.x xbmc binding.
 
 ## Preparation
 
-In order to allow control of Kodi by this binding, you need to enable the Kodi application remote control feature.
```

Comment:
One minor question:

What was wrong with this part?

The new sentence is definitely wrong. The correct spelling should be "controlled by" or "controlled via".

Reply:
The sentence seemed a little awkward to me, thanks for catching the mistake.

Reply:
> "controlled via".
>
Seems perfect here